tests/implementor/test_table_run.py

```diff
@@ -37,20 +48,37 @@ class TestRunSuccessfulInit:
     def setup_method(self):
         pass
 
+    @pytest.fixture
+    def token(self):
+        token = Token(name="owner", permission="admin", hashedToken="hash")
```

Comment:
Should we add validation to prevent not hashed tokens from being entered in the database?

Reply:
Is it possible to distinguish between a random string and a hash other than prepending `hash`? 🤔 

IMO it's not needed because we insert the tokens by ourselves. 🙃 We'll generate the token, save a hashed version in the database and return the original to the user.

Reply:
Well if you look at (for example) a sha256 hash, it will look consistent (64 hexadecimal chars). If one were to add salt & pepper to the hash, the format would get even more obvious.
In any case, setting a parameter called `hashedToken` to something obviously not hashed, seems a little off to me :slightly_smiling_face: